examples/simple-asset-transfer/package.json

```diff
@@ -2,13 +2,12 @@
 	"version": "0.1.0",
 	"name": "@hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer",
 	"dependencies": {
-		"@hyperledger-labs/blockchain-integration-framework": "git+https://github.com/hyperledger-labs/blockchain-integration-framework.git#fix/peter.somogyvari/make-ci-pass",
+		"@hyperledger-labs/blockchain-integration-framework": "../../.tmp/hyperledger-labs-blockchain-integration-framework-dev.tgz",
```

Comment:
Usual directory name on most operating system's for temporary files, it's a convention I like due to personal preference. Didn't want to use `build` because I thought it best to reserve that for the artifacts that end up in a release, but the package archive I'm storing in the .tmp directory is not intended to be part of a release of the software.